Makefile

```diff
@@ -92,10 +92,10 @@ test_go_race:
 	$(go_test) ./... -race
 
 test_sharness_short:
-	cd test/sharness/ && make
+	make -C test/sharness/
 
 test_sharness_expensive:
-	cd test/sharness/ && TEST_EXPENSIVE=1 make
+	TEST_EXPENSIVE=1 make -C test/sharness/
```

Comment:
@chriscool was there a reason to do the `make` after a `cd`? pls confirm this change is perfectly fine with you.